back/api/migrations/0033_auto_20211221_1431.py

```diff
@@ -0,0 +1,91 @@
+# Generated by Django 3.2.8 on 2021-12-21 13:31
+
+import django.contrib.gis.db.models.fields
+import django.contrib.gis.geos.polygon
+from django.db import migrations, models
+import django.db.models.deletion
+import djmoney.models.fields
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api', '0032_auto_20211221_1223'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='metadata',
+            name='description_short',
+        ),
+        migrations.AlterField(
```

Comment:
Are these currency definition here on purpose?

Reply:
It's because the django-money package got updated and I didn't run the migrations directly after the update